pkg/clusteragent/clusterchecks/dispatcher_configs.go

```diff
@@ -89,13 +89,19 @@ func (d *dispatcher) removeConfig(digest string) {
 	defer d.store.Unlock()
 
 	node, found := d.store.getNodeStore(d.store.digestToNode[digest])
+
+	checkName := ""
+	if found {
+		checkName = node.digestToConfig[digest].Name
```

Comment:
I noticed that `node` is `Lock`ed a little bit further in this file. At line 111. So, I’m wondering if it should also be `Lock`ed here.

Reply:
Added :+1: